documentation/book/ref-operators-cluster-operator-configuration.adoc

```diff
@@ -66,3 +66,7 @@ if no image is specified as the `Kafka.spec.entityOperator.userOperator.image` i
 `STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE`:: Optional, default `strimzi/entity-operator-stunnel:latest`.
 The image name to use as the default when deploying the sidecar container which provides TLS support for the Entity Operator, if
 no image is specified as the `Kafka.spec.entityOperator.tlsSidecar.image` in the xref:assembly-configuring-container-images-deployment-configuration-kafka[].
+
+`STRIMZI_VOLUME_MOUNT_INIT_IMAGE`:: Optional, default `busybox`.
+The image to use as default for the init container required for volume mounting on kubernetes.
+If a persistent volume claim is requested and the running cluster is a Kubernetes one there is an hack on volume mounting which needs an "init-container".
```

Comment:
Maybe we should explain better what this hack does?

Reply:
Thanks for the suggestion & quick reply. Do you need any more details?